chia/data_layer/data_layer_wallet.py

```diff
@@ -1272,10 +1272,9 @@ async def finish_graftroot_solutions(offer: Offer, solver: Solver) -> Offer:
                         asserted_root: Optional[str] = None
                         proofs_of_inclusion = []
                         for value in values:
-                            for root in solver["proofs_of_inclusion"].info:
-                                proof: Tuple[int, List[bytes32]] = tuple(  # type: ignore
-                                    solver["proofs_of_inclusion"][root]
-                                )
+                            for proof_of_inclusion in solver["proofs_of_inclusion"]:
+                                root = proof_of_inclusion[0]
+                                proof: Tuple[int, List[bytes32]] = tuple(proof_of_inclusion[1:])
```

Comment:
Is the `str` correct here?  Or would it have been processed into a `bytes32`?  Or would that depend on it having a leading `0x` or not?

```suggestion
                                root: str = proof_of_inclusion[0]
                                proof: Tuple[int, List[bytes32]] = (proof_of_inclusion[1], proof_of_inclusion[2])
```

Reply:
Yeah str is correct right now